atlassian-terraform/atlassian.tf

```diff
@@ -25,55 +25,55 @@ resource "restapi_object" "ops_to_sumo_webhook" {
   path  = "/v2/integrations"
   # debug = true
   id_attribute = "data/id"
-  data         = "${data.template_file.data_json[0].rendered}"
+  data         = data.template_file.data_json[0].rendered
```

Comment:
so we don't need ${} any more?

Reply:
Yes and No. Basically, if it is a variable only assignment then brackets are not required, for example from Terraform docs: an expression like "${foo}" should be rewritten as just foo.

However, if the variable is used in a complex expression involving other variables and strings then these are required as Terraform cannot differentiate between variables and strings, examples from our script: url = "${var.jira_server_url}/rest/api/2/issue"
template = "${file("${path.module}/templates/sumo_to_jiracloud_webhook.json.tmpl")}"

Attached is the warning from Terraform itself which explains the same.
<img width="569" alt="Screen Shot 2020-05-21 at 10 04 46 AM" src="https://user-images.githubusercontent.com/10919488/83934948-28574480-a7d2-11ea-8a63-addc1e6e953c.png">

Reply:
Cool then